src/shared/components/person/profile.tsx

```diff
@@ -57,6 +59,7 @@ interface ProfileState {
   sort: SortType;
   page: number;
   loading: boolean;
+  personBlocks: PersonBlockView[];
```

Comment:
The entire array of blocks isn't really needed here. All that's needed is either a boolean for `personBlocked`, or a single `personBlockView`